src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/CSharp/ScenarioRunResult.cs

```diff
@@ -81,6 +83,11 @@ public ScenarioRunResult(
     {
     }
 
+    /// <summary>
+    /// Gets or sets the Version of the format used to persist this scenario run. You should not modify this value from the default.
+    /// </summary>
+    public int FormatVersion { get; init; } = formatVersion;
```

Comment:
We previously made the call to make the reporting types completely mutable. I think it would be best to stick to that and make this property mutable too to match everything else.

Reply:
I don't have a strong opinion either way, but happy to roll it back.